modules/http-api/src/main/java/org/mule/runtime/http/api/sse/ServerSentErrorContext.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.sse;
+
+/**
+ * Error context. It contains the exception and... TODO: What else do we want here?
```

Comment:
Some reference to the SSE connection lifecycle (e.g: if the error happened during the initiator request, or once the long polling was already streaming, etc)